src/plugins/intel_cpu/src/nodes/executors/acl/acl_deconv.hpp

```diff
@@ -0,0 +1,193 @@
+
+    impl_desc_type getImplType() const override {
+        return implType;
+    }
+
+private:
+    DeconvAttrs deconvAttrs;
+    impl_desc_type implType = impl_desc_type::acl;
+
+    arm_compute::Tensor srcTensor;
+    arm_compute::Tensor weiTensor;
+    arm_compute::Tensor biasTensor;
+    arm_compute::Tensor dstTensor;
+    std::unique_ptr<arm_compute::NEDeconvolutionLayer> deconv = nullptr;
+};
+
+class AclDeconvExecutorBuilder : public DeconvExecutorBuilder {
+public:
+    static bool customIsSupported(const DeconvAttrs& deconvAttrs,
+                                  const std::vector<MemoryDescPtr>& srcDescs,
+                                  const std::vector<MemoryDescPtr>& dstDescs) {
+        if ((srcDescs[0]->getShape().getDims().size() != 3 && srcDescs[0]->getShape().getDims().size() != 4) ||
+             dstDescs[0]->getShape().getDims().size() != srcDescs[0]->getShape().getDims().size() ||
+             srcDescs[1]->getShape().getDims().size() != 4) {
+            DEBUG_LOG("AclDeconvExecutor does not support dimension:",
+                      " src[0]=", srcDescs[0]->getShape().getDims().size(),
+                      " src[1]=", srcDescs[1]->getShape().getDims().size(),
+                      " dst[0]=", dstDescs[0]->getShape().getDims().size());
+            return false;
+        }
+
+        // TODO: Ticket CVS-114087 - enable FP16 when check FP16 scoup
+        if (!(one_of(srcDescs[0]->getPrecision(), /*InferenceEngine::Precision::FP16, */InferenceEngine::Precision::FP32) &&
+              srcDescs[0]->getPrecision() == srcDescs[1]->getPrecision() &&
+              srcDescs[1]->getPrecision() == dstDescs[0]->getPrecision())) {
+            DEBUG_LOG("AclDeconvExecutor does not support precisions:",
+                      " src[0]=", srcDescs[0]->getPrecision(),
+                      " src[1]=", srcDescs[1]->getPrecision(),
+                      " dst[0]=", dstDescs[0]->getPrecision());
+            return false;
+        }
+
+        if (deconvAttrs.withBiases && srcDescs[2]->getPrecision() != srcDescs[0]->getPrecision()) {
+            DEBUG_LOG("AclDeconvExecutor does not support precisions:",
+                      " src[2]=", srcDescs[2]->getPrecision());
+            return false;
+        }
+
+        if (!(srcDescs[0]->hasLayoutType(LayoutType::ncsp) &&
+              srcDescs[1]->hasLayoutType(LayoutType::ncsp) &&
+              dstDescs[0]->hasLayoutType(LayoutType::ncsp)) &&
+            !(srcDescs[0]->hasLayoutType(LayoutType::nspc) &&
+              srcDescs[1]->hasLayoutType(LayoutType::nspc) &&
+              dstDescs[0]->hasLayoutType(LayoutType::nspc))) {
+            DEBUG_LOG("AclDeconvExecutor does not support layouts:",
+                      " src[0]=", srcDescs[0]->serializeFormat(),
+                      " src[1]=", srcDescs[1]->serializeFormat(),
+                      " dst=", dstDescs[0]->serializeFormat());
+            return false;
+        }
+
+        if (deconvAttrs.withBiases &&
```

Comment:
in this code block it is enough to check only bias layout.

Reply:
@dmitry-gorokhov corrected